tests/config.js

```diff
@@ -16,19 +16,19 @@ module.exports = class Config {
             devicedb_prefix: \"vagrant\"
             devicedb_bucket: \"lww\"
             relay_id: \"{{device_id}}\"
-            ca_chain: \"$DEVICEDB_SRC/hack/certs/myCA.pem\"
+            ca_chain: \"/home/vagrant/work/gostuff/src/github.com/armPelionEdge/devicedb/hack/certs/myCA.pem\"
```

Comment:
Heh, `gostuff`

Reply:
Don't like how it is hard coded... But for now, this needs to match the vagrant VM path 🤣